tests/conftest.py

```diff
@@ -116,7 +116,7 @@ def test_with_any_microvm(test_microvm_any):
 session root and temporary directories.
 """
 
-DEFAULT_ROOT_TESTSESSION_PATH = '/tmp/firecracker_test_session/'
+DEFAULT_ROOT_TESTSESSION_PATH = '/srv/tmp/firecracker_test_session/'
```

Comment:
Why does this have to change?

Reply:
When running block tests the dev container needs to be started with -r and that will create a ramdisk where the block device file lives. 
So this change actually makes this possible to have the temporary block device filesystem stay in the ramdisk.